### PR TITLE
Cap output voltage at 1.1 * input voltage

### LINKED ISSUE #70: Can set output voltage higher than possible output
Is the change made to solve #52 correct? It seems like we want to limit the settable output to what the device can actually output so around 90-95% of the input voltage

### PR DESCRIPTION
**Pull request by Xenoamor**
Fixes #70

### CODE REVIEW

File: opendps/func_cv.c [hunk 1]
```diff
@@ -285,7 +285,7 @@ static void cv_tick(void)
     uint16_t i_out_raw, v_in_raw, v_out_raw;
     hw_get_adc_values(&i_out_raw, &v_in_raw, &v_out_raw);
     /** Continously update max voltage output value */
-    cv_voltage.max = pwrctl_calc_vin(v_in_raw);
+    cv_voltage.max = (float) pwrctl_calc_vin(v_in_raw) / 1.1f + 0.5f; /** Max output voltage is 1.1 * Vin */
```
[the remark this answers is not in the document]
Comment (Xenoamor) [reply opendps/func_cv.c]: Hmm see that's what I was thinking but I looked at [this page here](https://www.banggood.com/DPS5005-50V-5A-Buck-Adjustable-DC-Constant-Voltage-Power-Supply-Module-Integrated-Voltmeter-Ammeter-p-1062473.html?akmClientCountry=GB&cur_warehouse=CN) and they say that for the following models:
DP20V2A, DP30V3A, DP50V2A, DP50V5A, DPS3003, DPS3005, DPS5005, DPS3012, DPS5015

> You must make sure that the input voltage is 1.1 times higher than output voltage

File: opendps/func_cv.c [hunk 2]
```diff
@@ -285,7 +285,7 @@ static void cv_tick(void)
     uint16_t i_out_raw, v_in_raw, v_out_raw;
     hw_get_adc_values(&i_out_raw, &v_in_raw, &v_out_raw);
     /** Continously update max voltage output value */
-    cv_voltage.max = (float) pwrctl_calc_vin(v_in_raw) / 1.1f + 0.5f; /** Max output voltage is 1.1 * Vin */
+    cv_voltage.max = (float) pwrctl_calc_vin(v_in_raw) / VIN_VOUT_RATIO + 0.5f; /** Max output voltage is 1.1 * Vin */
```
Comment (kanflo) [inline opendps/func_cv.c]: Oh, and one more thing. If we have devised the ratio, why add another 0.5V? Just curious...

Comment (Xenoamor) [reply opendps/func_cv.c]: When we typecast from a float to a unsigned integer the value after the decimal point is lost i.e. 4.7 will become 4. This is obviously wrong as we would rather it gets rounded to 5. By adding 0.5 to it we will always round correctly

This only works for positive numbers however as cv_voltage.max is unsigned we are safe to make this assumption

Comment (Xenoamor) [reply opendps/func_cv.c]: I made some similar changes [on my repo over here](https://github.com/Xenoamor/opendps/commit/adbd69af11f66b9347dc27b3d2e50535e2ae5bfc)

Comment (kanflo) [reply opendps/func_cv.c]: Right! Could you please add that to the comment which I now see should read "Max output voltage is Vin / 1.1"

Comment (Xenoamor) [reply opendps/func_cv.c]: Done and done. Well spotted!
`cv_voltage.max` is actually a signed int but `pwrctl_calc_vin()` returns a unsigned int so we should never have a negative float value and this rounding method will still work

**Pull request merged**